pyproject.toml

```diff
@@ -58,18 +60,33 @@ filterwarnings = [
     "error",
 
     # Warnings raised by dependencies of dependencies, out of our control
+    # https://github.com/Cadene/pretrained-models.pytorch/issues/221
+    "ignore:.* is deprecated and will be removed in Pillow 10:DeprecationWarning:pretrainedmodels.datasets.utils",
     # https://github.com/pytorch/vision/pull/5898
+    "ignore:.* is deprecated and will be removed in Pillow 10:DeprecationWarning:torchvision.transforms.functional_pil",
     # https://github.com/rwightman/pytorch-image-models/pull/1256
-    "ignore:.*is deprecated and will be removed in Pillow 10:DeprecationWarning",
```

Comment:
If we don't specify a module, this will ignore the test for all files, including our own. By specifying a model, we can ignore things with more fine-grained precision. We should probably do this for all warnings raised by deps of deps.